spec/features/managing_items_and_wishlists_spec.rb

```diff
@@ -33,6 +33,26 @@
       expect(page).to have_text "Needed: 18"
     end
 
+    scenario "I get redirected to the search page when Amazon returns an error code.", :external do
+      allow_any_instance_of(HTTParty::Response).to receive(:code).and_return(500)
```

Comment:
Rspec docs say that `allow_any_instance_of` is for legacy apps. Is it possible to update this to use the newer `allow` syntax?

Reply:
The problem is that HTTParty::Response is really hard to fake. It's a huge object, and it's not obvious what attributes need to be present. I can't immediately think of a way to get a reference to the response, so I'll have to take a longer look at this.